android/assets/jsons/translations/German.properties

```diff
@@ -5835,11 +5835,10 @@ The Maya measured time in days from what we would call 11th of August, 3114 BCE.
 Unciv only displays ය B'ak'tuns, ඹ K'atuns and ම Tuns (from left to right) since that is enough to approximate gregorian calendar years. The Maya numerals are pretty obvious to understand. Have fun deciphering them! = Unciv zeigt nur ය B'ak'tuns, ඹ K'atuns und ම Tuns (von links nach rechts) an, da dies ausreicht, um gregorianische Kalenderjahre anzunähern. Die Maya-Ziffern sind ziemlich einfach zu verstehen. Viel Spaß beim Entschlüsseln!
 
  # Requires translation!
-Your cities will periodically demand different luxury goods to satisfy their desire for new things in life. = 
+Your cities will periodically demand different luxury goods to satisfy their desire for new things in life. = Eure Städte werden wiederholend verschiedene Luxusgüter fordern, welche deren Verlangen nach neuen Dingen erfüllen.
```

Comment:
Hier bitte wieder "Du" verwenden
"in regelmäßigen Abständen" passt aus meiner Sicht besser als "wiederholend". Was meinst du?